README.md

```diff
@@ -68,7 +69,11 @@ p(thing, {
       structure: s(structure =>
         structure.concat('roflmao') // Why not
       )
-    })
+    }),
+    with: ps(
+      [],
+      {1: 'copy'}
+    ) // ['a', 'copy', 'tacked'], the original array is left untouched
```

Comment:
The fact this incidentally demonstrates you don't need `scope` to extend arrays detracts from the feature a bit, especially seeing as this all-in-one example is mutative at the top level, so the fact the original array is preserved isn't easy to intuit on an intent level. The docs already needed a re-write but this brings the necessity closer.

Reply:
> The fact this incidentally demonstrates you don't need `scope` to extend arrays

Only when you know the indices in advance. This is mostly useful for tuples.